Include/Private/MyPageGenerator/MainWindow.H

```diff
@@ -51,6 +54,14 @@ private:
   void generateMenuBar();
   void generateFileMenu();
 
+  static QSharedPointer<QSettings> CreateQSettings();
```

Comment:
I don't want to use STL stuff in the marsh of Qt and I tried `QScopedPointer` but it is much like `std::auto_ptr` I guess.